src/index.ts

```diff
@@ -1,24 +1,51 @@
 import { ensureString } from "ensure-string";
 import mlArrayMax from "ml-array-max";
-import uniqueXFunction from "ml-arrayxy-uniquex";
-import { xIsMonotone } from "ml-spectra-processing";
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const uniqueXFunction = require("ml-arrayxy-uniquex");
```

Comment:
please convert the `require` to `import`

Reply:
ml-arrayxy-uniquex is not typescript package using import cause problems